extension/lib/BiasDoorhanger.jsm

```diff
@@ -68,7 +68,7 @@ class BiasDoorhanger {
 
   hideForever() {
     const hostname = Hosts.getHostnameFromURI(this.focusedURI);
-    CommonStorage.put(-1, hostname);
+    DoorhangerStorage.put(-1, hostname);
```

Comment:
Shouldn't this be `{timestamp: Date.now(), neverAgain: true}` instead of `-1` now?